etcdserver/etcdserverpb/rpc.proto

```diff
@@ -658,6 +664,23 @@ message LeaseKeepAliveResponse {
   int64 TTL = 3;
 }
 
+message LeaseTimeToLiveRequest {
+  // ID is the lease ID for the lease to keep alive.
+  int64 ID = 1;
+}
+
+message LeaseTimeToLiveResponse {
+  ResponseHeader header = 1;
+  // ID is the lease ID from the keep alive request.
+  int64 ID = 2;
+  // TTL is the new time-to-live for the lease.
+  int64 TTL = 3;
+  // Remaining is the remaining time in seconds before expiration.
+  int64 remaining = 4;
+  // Keys is the list of keys attached to this lease.
```

Comment:
Ok will remove `remaining`. I think `remaining` can be useful with longer TTL and user wants to know what's the remainder. Ok we can easily add later.